File: Code/C#/HVPSAPI/Messages/Message.cs
using Core.Messages.Messages;
using HVPSAPI.DataMemberNames.Messages;
using System.Runtime.Serialization;
using System.Text.Json.Serialization;

namespace HVPSAPI.Messages
{
    [DataContract]
    public class Message : TypedMessageBase
    {
        [JsonPropertyName(MessageDataMemberNames.Content)]
        [JsonInclude]
        [DataMember(Name = MessageDataMemberNames.Content)]
        public string Content { get; protected set; }
        public Message()
            : base()
        {
            Type = MessageTypes.Message;
        }
    }
}

File: Code/JavaScript/client/src/messages/Message.js
export default class Message
{
 static toJSON(o){ 
    const r = {};
   r["tpe"]="m";
    r["c"]=o.content;
    return r;
 }
 static fromJSON(o){
    const r = {};
    r.content=o["c"];
   return r;
 }
}